src/js/python/tree.js

```diff
@@ -27,6 +27,18 @@ export default {
                 ],
                 monitors: [monitors.standard_output],
             },
+            "processing:find_extrema": {
```

Comment:
Using the example from https://github.com/Exabyte-io/exabyte-application-flavors/pull/58/files#diff-64ab3b8d30463abea4548a02aebf3802c09f7c0373af29d6d9b703c8532b6658R73 (`pyml:data_input:train_test_split:sklearn`) this should be:

`python:processing:find_extrema:scipy`

Reply:
I would not preface with `python`, as it is already evident by choosing a python flavor. Perhaps we can leave it empty (`:processing:find_extrema:scipy`) or use a more specific term (`algorithms:processing:find_extrema:scipy` or `numeric:processing:find_extrema:scipy`).

Reply:
Maybe `generic:processing:find_extrema:scipy`?